js/Items.js

```diff
@@ -0,0 +1,99 @@
+angular.module('PathOfDamage')
```

Comment:
I don't think its an issue. A getter is good styling and the rest of the indents are angular artifacts.